### PR TITLE
Add CSS-in-JavaScript section

### PR DESCRIPTION
**Pull request by lencioni**
We have been writing CSS in JavaScript and have developed a set of best
practices around this type of thing. I thought it would be nice to
publish this document alongside our other JavaScript style guide
documents.

### CODE REVIEW

File: css-in-javascript/README.md [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+      desktop: '@media (min-width: 1048px)',
+      tablet: '@media (max-width: 1047px)',
+      mobile: '@media (max-width: 639px)',
+    };
+
+    // good
+    const breakpoints = {
+      large: '@media (min-width: 1048px)',
+      medium: '@media (max-width: 1047px)',
+      small: '@media (max-width: 639px)',
```
Comment (frenzzy) [inline css-in-javascript/README.md]: How about order? Mobile first? :)

Comment (lencioni) [inline css-in-javascript/README.md]: Sure! 9fc05bf

**Comment by ButuzGOL** [comment]
👍

File: css-in-javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+  - Namespace custom theme settings under a nested object with a unique and descriptive key.
+
+    ```js
+    // bad
+    ThemedStyleSheet.registerTheme('mySection', {
+      mySectionPrimaryColor: 'green',
+    });
+
+    // good
+    ThemedStyleSheet.registerTheme('mySection', {
```
Comment (ljharb) [inline css-in-javascript/README.md]: There's no explanation in this document about `ThemedStyleSheet`, nor where it comes from.

Comment (lencioni) [inline css-in-javascript/README.md]: Good call. I'll add a sentence to the first bullet point in this section.

Comment (lencioni) [inline css-in-javascript/README.md]: c640633

**Pull request merged**